app/screens/channel/channel_post_list/channel_post_list.js

```diff
@@ -58,6 +58,15 @@ export default class ChannelPostList extends PureComponent {
         this.contentHeight = 0;
     }
 
+    componentDidMount() {
+        this.mounted = true;
```

Comment:
is there a reason for this?

Reply:
Yes, I was getting some setState on unmounted component warnings from this:
https://github.com/uber-uchat/mattermost-mobile/blob/ca1659c65835eb0c3a0d632811eaa860c1585f73/app/screens/channel/channel_post_list/channel_post_list.js#L63